blender-cli/src/withdraw.rs

```diff
@@ -86,6 +83,19 @@ pub(super) fn do_withdraw(
         recipient_bytes,
     );
 
+    let pk = match fs::read(proving_key_file) {
```

Comment:
I think you could extract it out into a utility parametrized by `deposit/withdraw` but it's a nitpick :)